fontir/src/error.rs

```diff
@@ -14,9 +14,9 @@ pub enum Error {
     /// A source file was not understood
     #[error(transparent)]
     BadSource(#[from] BadSource),
-    /// An error occured while attempting to track a file
-    #[error(transparent)]
-    TrackFile(#[from] TrackFileError),
+    /// What path?!
+    #[error("{0:?} does not exist")]
```

Comment:
```suggestion
    #[error("{0} does not exist")]
```

debug is unnecessary, `thiserror` knows to special case this and call `Path::display`.

Reply:
fixed